pages/video/[id].tsx

```diff
@@ -62,28 +65,13 @@ export default function Video({ id }: ParsedUrlQuery) {
   const { user } = useAuth();
   const [notes, setNotes] = useState<INotes[] | []>([]);
   const noteInput = useRef<HTMLInputElement>(null);
```

Comment:
good to see you are using ref for inputs thereby using uncontrolled inputs. 

I have a great lib recommendation for you. React Hook Form. 

You can skip its usage here but learn it sometime

Reply:
Thanks for the advice. I surely look into it. Thank you.